### ISSUE TITLE
Look at documentation

### ISSUE THREAD

**Issue by MMT-LD** [Trufa-me/Node-ES6-API-Framework#3]
We need to look at the documentation and remove unnecessary detail, agree? 

looks linked to this commit: c2513d38a5156a834c143249ae4721af44900195

**Comment by okbrown** [comment]
Agreed. Just made some comments regarding `nodejs versions` when using` async functions`.

**Comment by MMT-LD** [comment]
Async functions came out in 7.6, however, we have babel running... so if we state the LTS version for now then we will probably get less questions around using the "stable" version.
